app/Http/Controllers/Api/V1/CurrikiGo/PublishController.php

```diff
@@ -99,19 +102,18 @@ public function playlistToMoodle(PublishMoodlePlaylistRequest $publishRequest, P
             $counter = (isset($data['counter']) ? intval($data['counter']) : 0);
 
             $moodle_playlist = new MoodlePlaylist($lmsSetting);
-            $response = $moodle_playlist->send($playlist, ['counter' => intval($counter)]);
+            $response = $moodle_playlist->send($playlist, ['counter' => intval($counter)], $gcrRepo);
 
             $code = $response->getStatusCode();
             if ($code == 200) {
-                $outcome = $response->getBody()->getContents();
                 return response([
-                    'data' => json_decode($outcome),
+                    'data' => json_decode($response->getBody()),
                 ], 200);
             }
 
             return response([
                 'errors' => ['Failed to send playlist to Moodle.'],
-            ], 500);
+            ], 400);
```

Comment:
why 400 ?

Reply:
because 500 shows API crash and in this case API is not crashed it just not creates playlist due to multiple reasons.